tests/screenshot/apps/components/Picker.js

```diff
@@ -32,6 +32,7 @@ const pickerList = {
 const PickerTests = [
 	<Picker>{[]}</Picker>,
 	<Picker>{['Hello', 'Hello']}</Picker>,
+	// Icon is not displayed (since it is invalid value)
 	<Picker decrementIcon incrementIcon>hello</Picker>,
```

Comment:
How about add one more case for nomal operation?

```suggestion
	// Icon is not displayed (since `decrementIcon` and `incrementIcon` prop should be icon string)
	<Picker decrementIcon incrementIcon>hello</Picker>,
	<Picker decrementIcon="minus" incrementIcon="plus">hello</Picker>,
```

Reply:
I think nomal operation are already tested in horizontal picker(line 72, 252). So, how about add it in only vertical picker?

Reply:
Hm.. OK Let's add  a test for vertical picker.

Reply:
Good